juriscraper/opinions/united_states/state/lactapp_2.py

```diff
@@ -0,0 +1,147 @@
+        "West Carroll": "Fifth Judicial District Court for the Parish of West Carroll, Louisiana",
+        "East Carroll": "Sixth Judicial District Court for the Parish of East Carroll, Louisiana",
+        "Caddo Juvenile Court": "Juvenile Court for the Parish of Caddo, Louisiana",
+        "Caddo Shreveport City Court": "Shreveport City Court for the Parish of Caddo, Louisiana",
+        "DeSoto OWC District 1-W": "Office of Workers' Compensation District 1-W for the Parish of DeSoto, Louisiana",
+        "Lincoln Ruston City Court": "Ruston City Court for the Parish of Lincoln, Louisiana",
+        "Ouachita West Monroe City Court": "West Monroe City Court for the Parish of Ouachita, Louisiana",
+        "OUACHITA Monroe City Court": "Monroe City Court for the Parish of Ouachita, Louisiana",
+        "Franklin OWC District 1-E": "Office of Workers' Compensation District 1-E for the Parish of Franklin, Louisiana",
+        "Minden City Court Webster": "Minden City Court for the Parish of Webster, Louisiana",
+        "Morehouse Bastrop City Court": "Bastrop City Court for the Parish of Morehouse, Louisiana",
+        "Morehouse OWC District 1-E": "Office of Workers' Compensation District 1-E for the Parish of Morehouse, Louisiana",
+        "Webster Minden City Court": "Minden City Court for the Parish of Webster, Louisiana",
+        "Winn OWC District 2": "Office of Workers' Compensation District 2 for the Parish of Winn, Louisiana",
+    }
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        self.base_url = "https://www.la2nd.org/opinions/"
+        self.year = datetime.now().year
+        self.url = f"{self.base_url}?opinion_year={self.year}"
+        self.status = "Published"
+        self.target_date = None
+        self.make_backscrape_iterable(kwargs)
+
+    def _download(self):
+        html = super()._download()
+        # Currenly there are no opinions for 2025, so we need to go back one year
+        if html is not None:
+            tables = html.cssselect("table#datatable")
+            if not tables or not tables[0].cssselect("tbody tr"):
+                self.year -= 1
+                self.url = f"{self.base_url}?opinion_year={self.year}"
+                return self._download()
+        return html
+
+    def _process_html(self):
+        if self.html is None:
+            return
+
+        tables = self.html.cssselect("table#datatable")
+        if not tables or not tables[0].cssselect("tbody tr"):
+            return
+
+        logger.info(f"Processing cases for year: {self.year}")
+        for row in tables[0].cssselect("tbody tr"):
+            case_date = datetime.strptime(
+                get_row_column_text(row, 1), "%m/%d/%Y"
+            ).date()
+
+            if self.skip_row_by_date(case_date):
+                continue
+
+            author = get_row_column_text(row, 4)
+            clean_author = self.clean_judge_name(author)
+
```

Comment:
I appreciate the clean author method but you should just chuck it.  we have tools to help clean up judge strings. I think 

`author_str = get_row_column_text(row, 4)`
`normalize_judge_string(author_str)[0]` will most likely solve it - and it will make the judge name nice and neat.

Reply:
from juriscraper.lib.judge_parsers import (
    normalize_judge_string,
)